game.py

```diff
@@ -30,7 +24,67 @@ def reset(self):
         self.score = 0
         self.speed_boost_timer = 0
         self.bomb_eater_timer = 0
-        self.state = 'playing'
+        self.state = 'menu'
+
+    def draw_text(self, text, x, y, color, size=36, centered=True):
+        font = pygame.font.SysFont(None, size)
+        rendered = font.render(text, True, color)
+        rect = rendered.get_rect()
+        if centered:
+            rect.centerx = x
+        else:
+            rect.x = x
+        rect.y = y
+        screen.blit(rendered, rect)
+        return rect
+
+    def main_menu(self):
+        events = pygame.event.get()
+        screen.fill((30, 30, 30))
+        self.draw_text("The Snake Game", WIDTH // 2, HEIGHT * 1/4, WHITE, size=48)
+
+        start_btn = self.draw_text("Start Game", WIDTH // 2, HEIGHT // 2, WHITE)
+        settings_btn = self.draw_text("Settings", WIDTH // 2, HEIGHT * 2/3, WHITE)
+        quit_btn = self.draw_text("Quit", WIDTH // 2, HEIGHT * 3/4, WHITE)
+
+        mouse = pygame.mouse.get_pos()
+        clicked = any(event.type == pygame.MOUSEBUTTONDOWN for event in events)
+
+        pygame.display.flip()
+
+        if start_btn.collidepoint(mouse) and clicked:
+            self.state = 'playing'
+            return 'playing'
+        elif settings_btn.collidepoint(mouse) and clicked:
+            self.state = 'settings'
+            return 'settings'
+        elif quit_btn.collidepoint(mouse) and clicked:
+            pygame.quit()
+            sys.exit()
+
+    def settings(self):
+        events = pygame.event.get()
+        screen.fill(BLACK)
+        self.draw_text("Settings", WIDTH // 2, HEIGHT * 1/4, WHITE, size=48)
+
+        start_btn = self.draw_text("Start Game", WIDTH // 2, HEIGHT // 2, WHITE)
+        settings_btn = self.draw_text("Settings", WIDTH // 2, HEIGHT * 2/3, WHITE)
```

Comment:
I'm surprised that we have these two options here. You're already in the settings menu and maybe you should have to go back to click "Start Game"